src/Controls/src/Core/Properties/AssemblyInfo.cs

```diff
@@ -53,6 +53,8 @@
 [assembly: InternalsVisibleTo("CommunityToolkit.Maui.Markup")]
 [assembly: InternalsVisibleTo("CommunityToolkit.Maui.Markup.UnitTests")]
 
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
Is this OK? I had to do this to make it so `NSubstitue` would work

Reply:
Are you subbing for internal interfaces?

Reply:
Yea, it's substituting for the `IControlsView` that I added in this PR. I could see us making `IControlsView` public for NET8. 

I just wasn't sure if having this IVT on our released binaries would be a bad move. 

https://github.com/dotnet/maui/pull/12259/files#diff-59f9b98a863b084d3fcfc59e478166aa991104ce8d2dd1eb05af8d9cc20b018eR14